src/configuration/interfaces.ts

```diff
@@ -5,15 +5,28 @@ export interface FilterOption {
 
 export interface ReporterOption {
     name: string;
-    output? : string | Buffer;
+    output?: string | Buffer;
 }
 
 export interface Dictionary<T> {
     [key: string]: T;
 }
 
+export interface SkipJsErrorsOptions {
+    stack?: string | RegExp;
+    message?: string | RegExp;
+    pageUrl?: string | RegExp;
+}
+
+export type SkipJsErrorsCallback = (opts?: { message: string; stack: string; pageUrl: string }) => boolean;
+
+export interface SkipJsErrorsCallbackOptions {
```

Comment:
I propose to rename as `SkipJsErrorsCallbackWithOptionsObject` here and all related method in code.